tests/mock_network/mod.rs

```diff
@@ -645,3 +607,50 @@ fn node_pause_and_resume() {
     poll_and_resend(&mut nodes);
     verify_invariant_for_all_nodes(&network, &mut nodes);
 }
+
+#[test]
+fn carry_out_parsec_pruning() {
```

Comment:
would it make sense to put this test in the same position as vote_prune was since it replaces it? (for history).

Reply:
ok. will move the test to that position.